### PR TITLE
Couch epi

### PR DESCRIPTION
**Pull request by iilyak**
This PR is exact copy of PR merged in ASF land

### CODE REVIEW

File: src/mango_httpd_handlers.erl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+-module(mango_httpd_handlers).
+
+-export([url_handler/1, db_handler/1, design_handler/1]).
+
+url_handler(_) -> no_match.
+
+db_handler(<<"_index">>)        -> fun mango_httpd:handle_req/2;
+db_handler(<<"_explain">>)      -> fun mango_httpd:handle_req/2;
+db_handler(<<"_find">>)         -> fun mango_httpd:handle_req/2;
```
Comment (eiri) [inline src/mango_httpd_handlers.erl]: Not a complain, just curious, what with the spacing here? Is it a convention for the handlers' definition?

**Comment by eiri** [comment]
Seems legit from my point of view, but will need a review from someone with deeper knowledge of cloudant's version of mango. CC: @tonysun83

File: src/mango_httpd_handlers.erl (hunk 1 above)
Comment (iilyak) [inline src/mango_httpd_handlers.erl]: It is not a convention and it is inconsistent as well. For example it is aligned in [chttpd](https://github.com/apache/couchdb-chttpd/blob/master/src/chttpd_httpd_handlers.erl#L17:L28) and it is not in [showroom](https://github.com/cloudant/showroom/blob/master/src/cloudant_httpd_handlers.erl#L6:L13).
I'm happy to remove extra white space.

**Comment by rnewson** [comment]
looks familiar! +1

**Pull request merged**